cypress/integration/single-pdf/settings.ts

```diff
@@ -0,0 +1,43 @@
+describe('Single PDF display settings', () => {
+  beforeEach(() => {
+    cy.loadPdf('/pdf');
+    cy.log('open the settings menu');
+    cy.findByRole('button', { name: 'Settings' }).click();
+  });
+
+  it('should have the default settings', () => {
+    cy.findByRole('radio', { name: 'Paginated' }).should('be.checked');
+    cy.findByRole('radio', { name: 'Scrolling' }).should('not.be.checked');
+  });
+
+  it('should not have font or theme display settings', () => {
+    cy.findByRole('radiogroup', { name: 'text font options' }).should(
+      'not.exist'
+    );
+    cy.findByRole('radiogroup', { name: 'reading theme options' }).should(
+      'not.exist'
+    );
+    cy.findByRole('button', { name: 'Decrease font size' }).should('not.exist');
+    cy.findByRole('button', { name: 'Increase font size' }).should('not.exist');
+  });
+
+  it('should zoom in and out', () => {
+    cy.findByText('Julius H. Schoeps')
+      .should('be.visible')
+      .should('have.css', 'font-size', '8.9176px');
```

Comment:
are we worried at all about this being brittle?

Reply:
I think if it's not causing issues we can leave it.  Usually when things are this precise it sets off "brittle!!!" flags in my head, but it also seems pretty easy to make more forgiving if we need to.